### ISSUE TITLE
Fails to install

### ISSUE THREAD

**Issue by aminya** [timothek/chocolatey-emclient#2]
It says it is already installed!

![image](https://user-images.githubusercontent.com/16418197/101517055-e2786600-3945-11eb-8880-91aeaa568bf6.png)

**Comment by timothek** [comment]
Hi @aminya, did you try to reboot the machine and start the installation again? The packages depends on an windows update (kb2999226) which get's installed beforehand. This package requires a reboot.

On which windows version are you?

**Comment by aminya** [comment]
No, I did not reboot. I am on the latest Win 10 beta, I don't think I need any updates to install. 

I have emClient installed before this, but I uninstalled it (fully). Then, I was trying to install it using choco, but I failed to do it.

**Comment by timothek** [comment]
Then please try to reboot your device. In my experience with the choco test environment, this could resolve the issue (at least for windows server systems, I have not used the latest Windows 10 Beta). Does the installation of the MSI-file downloaded from the eM-Client Homepage without choco work without errors?

Otherweise I am not able to help you any further. On all my devices the installation works and the error specifies that this is a generic error for your local environment and not an issue with the package itself. I'm afraid, that there's not much I can do.

**Comment by denverpilot** [comment]
I believe I can shed some light on this old ticket.  The 1603 is misleading.  What's happening is this, which should reproduce the error.
- Install em-client via chocolatey
- Let em-client upgrade itself to the next version
- Next choco upgrade (or install) will fail.  

The MSI provided by em-client does not allow install of the same version if already installed.

Unfortunately I accidentally did this, and don't really want to mess with an uninstall and reinstall to make chocolatey happy, so I just ignore that em-client fails at every choco upgrade all, for now.  I tried the various switch options to ignore the result code output from MSI, but choco still considers it a failed "install".

I doubt there's a better way to handle a 1603 because it's generic by nature and the only way to avoid this one way of triggering it would be to query the system for version number prior to running the MSI itself.

Hope that helps.
